pride_cv.obo

```diff
@@ -4488,6 +4488,32 @@ is_a: PRIDE:0000654 ! Olink instrument model
 created_by: deepti
 creation_date: 2024-01-30T15:55:50Z
 
+[Term]
+id: PRIDE:0000659
+name: Ion Mobility Spectrometry 
+def: "Ion Mobility Spectrometry " [PRIDE:PRIDE]
+is_a: PRIDE:0000551 ! Separation method
+created_by: deepti
+creation_date: 2024-04-08T21:55:20Z
+
+[Term]
+id: PRIDE:0000660
+name: Differential mobility spectrometry 
+def: "Differential mobility spectrometry " [PRIDE:PRIDE]
+is_a: PRIDE:0000551 ! Separation method
+created_by: deepti
+creation_date: 2024-04-08T21:55:20Z
+
+[Term]
+id: PRIDE:0000661
+name: SelexION differential mobility separation device 
```

Comment:
`Selexion mobility separation` The name should be smaller @deeptijk is better for the annotations, you can put the long text in the description.

Reply:
I agree to keep it small..but since this approach uses both differential and mobility, I had to add both. May be just "SelexION" and leave the whole line in definition.

Reply:
If you put SelexION is not clear if is the device or the separation method.

Reply:
ok..will add that